components/kyma-environment-broker/internal/model.go

```diff
@@ -35,13 +36,109 @@ const (
 )
 
 type CLSInstance struct {
-	Version                  int
-	ID                       string
-	GlobalAccountID          string
-	Region                   string
-	CreatedAt                time.Time
-	ReferencedSKRInstanceIDs []string
-	RemovedBySKRInstanceID   string
+	version         int
+	id              string
+	globalAccountID string
+	region          string
+	createdAt       time.Time
+	references      []string
+	removedBy       string
+
+	events []interface{}
+}
```

Comment:
I know everything is here (in this file) but maybe let's move CLSInstance to another go file

Reply:
Yeah, makes sense. Will do it

Reply:
Done